.github/workflows/integration.yml

```diff
@@ -424,15 +428,31 @@ jobs:
             docker logs $i >${name}.log
           done
 
+      - name: Get images and tags
+        id: images
+        run: |
+          hep_tag=${{ needs.backend_build.result == 'skipped' && 'master' || env.REF_TAG }}
+          editor_tag=${{ needs.editor_build.result == 'skipped' && 'master' || env.REF_TAG }}
+          ui_tag=${{ needs.ui_build.result == 'skipped' && 'master' || env.REF_TAG }}
```

Comment:
I would remove this and add:
`BACKEND_IMAGE`, `UI_IMAGE` and `EDITOR_IMAGE` to the **workflow** env. so we can reuse it in the steps where we start the containers.